cristin-commons/src/main/java/no/unit/nva/cristin/model/query/CristinCodeFacet.java

```diff
@@ -25,8 +25,4 @@ public Map<String, String> getLabels() {
         return name;
     }
 
-    @Override
-    public CristinFacetKey getCristinFacetKey() {
```

Comment:
an instance of CristinCodeFacet will not always map to SECTOR, see other comment. Also relevant for other models